applications/nrf_desktop/doc/ble_conn_params.rst

```diff
@@ -33,14 +33,21 @@ The option is implied by :ref:`CONFIG_DESKTOP_BT_CENTRAL <config_desktop_app_opt
 
 Enable :ref:`CONFIG_DESKTOP_BLE_USB_MANAGED_CI <config_desktop_app_options>` to manage Bluetooth connections' parameters reacting on the USB state change.
 The connection intervals for all of the Bluetooth connected peripherals are set to :ref:`CONFIG_DESKTOP_BLE_USB_MANAGED_CI_VALUE <config_desktop_app_options>` (100 ms by default) while USB is suspended.
-The connections' peripheral latencies are set to 0.
-The connection parameter change is reverted when USB is active.
+The connections' peripheral latencies are set to :ref:`CONFIG_DESKTOP_BLE_USB_MANAGED_LATENCY_VALUE <config_desktop_app_options>` (1 by default).
```

Comment:
```suggestion
The connections' peripheral latencies are set using the :ref:`CONFIG_DESKTOP_BLE_USB_MANAGED_LATENCY_VALUE <config_desktop_app_options>`  Kconfig option (``1`` by default).
```

Reply:
This seems inconsistent with the previous sentence. Rephrased a bit